contrib/apelscripts/condor_batch.sh

```diff
@@ -3,26 +3,24 @@
 # Create a temporary accounting file name
 today=$(date -u --date='00:00:00 today' +%s)
 yesterday=$(date -u --date='00:00:00 yesterday' +%s)
-output=batch-$(date -u --date='yesterday' +%Y%m%d )-$(hostname -s)
 
-# Build the filter for the history command
-CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
+OUTPUT_DIR="$(condor_ce_config_val APEL_OUTPUT_DIR)"
+OUTPUT_FILE="$OUTPUT_DIR/batch-$(date -u --date='yesterday' +%Y%m%d )-$(hostname -s)"
 
-function die() { echo "$0 aborted when reading config"; exit 1; }
-trap die ERR;
-if [ $# -gt 0 ]; then
-  source $1
+if [ ! -d $OUTPUT_DIR ] || [ ! -w $OUTPUT_DIR ]; then
+    echo "Cannot write to $OUTPUT_DIR"
+    exit 1
 fi
-trap '' ERR
 
-[[ -z "$SCALING_ATTR" ]] && SCALING_ATTR=MachineAttrRalScaling0
-[[ -z "$BATCH_HOST" ]] && BATCH_HOST=`hostname`
-[[ -z "$OUTPUT_LOCATION" ]] && OUTPUT_LOCATION=/var/lib/condor-ce/apel/
+# Build the filter for the history command
+CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
 
-OUTPUT_FILE=$OUTPUT_LOCATION/$output
+HISTORY_SUFFIX='-format "\n" EMPTY'
+SCALING_ATTR=$(condor_ce_config_val APEL_SCALING_ATTR)
+[ $? -eq 0 ] && HISTORY_SUFFIX="-format "%v|" ${SCALING_ATTR} ${HISTORY_SUFFIX}"
```

Comment:
Note that the `|` here is completely unquoted and will be interpreted as a pipeline to be executed in this line

```suggestion
[ $? -eq 0 ] && HISTORY_SUFFIX="-format \"%v|\" ${SCALING_ATTR} ${HISTORY_SUFFIX}"
```

Reply:
(Credit @matyasselmeci for catching this one)